backend/collaboration-service/app/src/helpers/rooms.helper.ts

```diff
@@ -1,12 +1,18 @@
 import { Text } from "@codemirror/state"
 import moment from "moment"
+import { DIFFICULTY, TOPIC_TAG } from "../constants/question"
+import { DEFAULT_LANGUAGE } from "../constants/language"
 
-export const generateNewDocument = (roomId: string) => {
+export const generateNewDocument = (roomId: string, difficulty: keyof typeof DIFFICULTY, topic: keyof typeof TOPIC_TAG) => {
     return {
         created: moment(),
         updated: moment(),
         updates: [],
         doc: Text.of([`Welcome to Room ${roomId}`]),
-        pending: []
+        pending: [],
+        difficulty,
```

Comment:
the difficulty + topic tag can be multi value
(e.g. easy + medium; string + array + two pointers, etc.)

Reply:
Resolved in [8304b6a](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/24/commits/8304b6ab4ab3e94be379ff3bea271b3f0f8d79b5)